Assets/Vimeo/Scripts/Config/VimeoVideo.cs

```diff
@@ -87,6 +88,9 @@ public VimeoVideo(JSONNode video)
 
                 // Sort the progressive files quality
                 for (int i = 0; i < files["progressive"].Count; i++) {
+                    if (!files["progressive"][i]["log"].IsNull) {
+                        playLoggingLink = GetPathFromUrl(files["progressive"][i]["log"].Value);
```

Comment:
since you are getting a path, then makes more sense to name the variable after path. `playLoggingPath`